api/base/views.py

```diff
@@ -75,3 +82,17 @@ def root(request, format=None):
             'users': absolute_reverse('users:user-list'),
         }
     })
+
+
+class OsfAPIViewMeta(type):
+    """Metaclass to ensure a base level of permissions are applied to all of our views
+    """
+    def __new__(cls, name, bases, attributes):
+        permission_classes = set((
+            drf_permissions.IsAuthenticatedOrReadOnly,
+            UserIsConfirmed,
+            UserIsNotDeactivated
+        ))
```

Comment:
In fact, anything in this set should be a default permission class in settings.py